src/utils/validations/login.js

```diff
@@ -31,9 +31,6 @@ export const lastName = (value) => {
 
 export const confirmPassword = (password, data) => {
   let helperText
-  if (password !== data.confirmPassword) {
-    helperText = 'common.errorMessages.passwordsDontMatch'
-  }
-  helperText = emptyField(password, helperText)
+  helperText = emptyField(password, password !== data.password ? 'common.errorMessages.passwordsDontMatch' : '')
```

Comment:
what is this? omg

Reply:
delete helper text and return this ternary, or with using if operator use few returns, 
if (password !== data.confirmPassword){
return error)
return result

Reply:
returned to previous variant but mentioninh helpertext as string